src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
```

Comment:
Sounds good. Turns out Chris wants this in build as well. Not so much for debugging, but more so that customers can easily discover the available variables. I'll move it into common in a subsequent PR.